code/game/machinery/engi_points_vendor.dm

```diff
@@ -217,23 +218,24 @@
 	exploding = TRUE
 	update_icon()
 	for(var/mob/M in player_list)
-		M << 'sound/machines/Alarm.ogg'
+		send_to_playing_players('sound/machines/Alarm.ogg')
 	var/turf/startpoint = get_turf(src)
 	sleep(100)
 	for(var/mob/M in player_list)
-		M << 'sound/effects/explosionfar.ogg'
+		send_to_playing_players('sound/effects/explosionfar.ogg')
```

Comment:
Do it how regular explosions do it! https://github.com/tgstation/tgstation/blob/master/code/game/objects/explosion.dm#L48